packages/@uppy/transloadit/src/Assembly.js

```diff
@@ -62,6 +65,64 @@ class TransloaditAssembly extends Emitter {
     this.close()
   }
 
+  #connectServerSentEvents () {
+    this.#sse = new EventSource(`${this.status.websocket_url}?assembly=${this.status.assembly_id}`)
+
+    this.#sse.addEventListener('open', () => {
+      // if server side events works, we don't need websockets anymore (it's just a fallback)
+      if (this.socket) {
+        this.socket.disconnect()
+        this.socket = null
+      }
+      clearInterval(this.pollInterval)
+      this.pollInterval = null
+    })
+
+    /*
+     * The event "message" is a special case, as it
+     * will capture events without an event field
+     * as well as events that have the specific type
+     * other event type.
+     */
+    this.#sse.addEventListener('message', (e) => {
+      if (e.data === 'assembly_finished') {
+        this.#onFinished()
+      }
+
+      if (e.data === 'assembly_uploading_finished') {
+        this.emit('executing')
```

Comment:
```suggestion
        this.emit('transloadit:assembly-executing')
```

Reply:
This is copied from https://github.com/transloadit/uppy/blob/5ec7187825c6af473a178a37196a5636c74072c5/packages/@uppy/transloadit/src/Assembly.js#L94-L96

Changing the event would be a breaking change, right?

Reply:
Gotta be careful here indeed, as some of the events are Assembly internal events, not Uppy Core event emitter, it's confusing.